heapster-saw/examples/mbox.sawcore

```diff
@@ -17,15 +17,32 @@ bv64_128 : Vec 64 Bool;
 bv64_128 = [False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,False,True,False,False,False,False,False,False,False];
 
 
-data Mbox : sort 0 where {
-  Mbox_nil : Mbox;
-  Mbox_cons : Vec 64 Bool -> Vec 64 Bool -> Mbox -> BVVec 64 bv64_128 (Vec 8 Bool) -> Mbox;
+-- An inductive type formulation of the Mbox type; this is just for
+-- documentation purposes, and isn't used in the below
```

Comment:
I think this accidentally a word:

```suggestion
-- documentation purposes, and isn't used in the code below
```

Reply:
"In the below" is a phase often used in papers as a synonym for "in the following"